### ISSUE TITLE
Upgrade snakeyaml to 1.31

### ISSUE THREAD

**Issue by sbouchet** [openshift/openshift-restclient-java#476]
see https://issues.redhat.com/browse/JBIDE-28634

**Comment by sbouchet** [comment]
also:
okhttp-4.9.3 -> okhttp-4.10.0
okio-2.8.0 -> okio-3.0.0
kotlin-stdlib-1.4.10 -> kotlin-stdlib-1.6.20
logging-interceptor-4.9.3 -> logging-interceptor-4.10.0

**Comment by adietish** [comment]
@sbouchet logging-interceptor isn't past of the dependencies of the client. okio, kotlin-stdlib are both transient deps that get bumped in the following way when I bump the dependencies in #477:
```
[INFO] com.openshift:openshift-restclient-java:jar:9.0.5-SNAPSHOT
[INFO] +- com.squareup.okhttp3:okhttp:jar:4.10.0:compile
[INFO] |  +- com.squareup.okio:okio-jvm:jar:3.0.0:compile
[INFO] |  |  +- org.jetbrains.kotlin:kotlin-stdlib-jdk8:jar:1.5.31:compile
[INFO] |  |  |  \- org.jetbrains.kotlin:kotlin-stdlib-jdk7:jar:1.5.31:compile
[INFO] |  |  \- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.5.31:compile
[INFO] |  \- org.jetbrains.kotlin:kotlin-stdlib:jar:1.6.20:compile
[INFO] |     \- org.jetbrains:annotations:jar:13.0:compile
[INFO] +- org.yaml:snakeyaml:jar:1.32:compile
[INFO] +- org.jboss:jboss-dmr:jar:1.6.1.Final:compile
[INFO] |  \- org.jboss.logging:jboss-logging:jar:3.4.2.Final:compile
[INFO] +- commons-io:commons-io:jar:1.3.2:compile
[INFO] +- org.apache.commons:commons-compress:jar:1.21:compile
[INFO] +- org.slf4j:slf4j-api:jar:2.0.1:compile
[INFO] +- commons-codec:commons-codec:jar:1.15:compile
[INFO] +- commons-lang:commons-lang:jar:2.6:compile
[INFO] +- org.apache.logging.log4j:log4j-api:jar:2.19.0:test
[INFO] +- org.apache.logging.log4j:log4j-core:jar:2.19.0:test
[INFO] +- org.apache.logging.log4j:log4j-slf4j-impl:jar:2.19.0:test
[INFO] +- junit:junit:jar:4.13.2:test
[INFO] |  \- org.hamcrest:hamcrest-core:jar:1.3:test
[INFO] +- org.mockito:mockito-core:jar:4.8.0:test
[INFO] |  +- net.bytebuddy:byte-buddy:jar:1.12.14:test
[INFO] |  +- net.bytebuddy:byte-buddy-agent:jar:1.12.14:test
[INFO] |  \- org.objenesis:objenesis:jar:3.2:test
[INFO] +- org.assertj:assertj-core:jar:3.23.1:test
[INFO] \- org.skyscreamer:jsonassert:jar:1.5.1:test
[INFO]    \- com.vaadin.external.google:android-json:jar:0.0.20131108.vaadin1:test
```

**Comment by sbouchet** [comment]
ok sounds good enough

**Comment by adietish** [comment]
@sbouchet released 9.0.5.Final: https://repository.jboss.org/nexus/service/local/repositories/releases/content/com/openshift/openshift-restclient-java/9.0.5.Final/openshift-restclient-java-9.0.5.Final.jar
